### PR TITLE
fix(rns): increases erc721 gas multiplier

### PR DESCRIPTION
**Pull request by jurajpiar**

### CODE REVIEW

File: src/contracts/rns/Marketplace.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ class MarketplaceContract extends ContractBase {
     txOptions: TransactionOptions,
   ): Promise<TransactionReceipt> {
     const buyTx = this.methods.buy(tokenId)
-    return this._send(buyTx, txOptions)
+    return this._call(buyTx, txOptions)
```
Comment (nduchak) [inline src/contracts/rns/Marketplace.ts:81]: Please revert this changes

Comment (jurajpiar) [reply src/contracts/rns/Marketplace.ts:81]: oh dear.. left it there by accident

**Review by nduchak — APPROVED** [review]
LGTM

**Review by itofarina — APPROVED** [review]
LGTM
